test/helpers/wait-for-update.ts

```diff
@@ -29,7 +30,7 @@ window.waitForUpdate = initialCb => {
           done.fail(e)
         }
       }
-      if (!hasError && !job.wait) {
+      if (!hasError && !job?.wait) {
```

Comment:
```suggestion
      if (!hasError && !job!.wait) {
```
Just to keep javascript code unchanged (also the other similar changes) pls